[2021-01-15 12:29:04 +0000] [21574] [INFO] Starting gunicorn 19.10.0
[2021-01-15 12:29:04 +0000] [21574] [INFO] Listening at: http://127.0.0.1:8000 (21574)
[2021-01-15 12:29:04 +0000] [21574] [INFO] Using worker: sync
[2021-01-15 12:29:04 +0000] [21749] [INFO] Booting worker with pid: 21749
[2021-01-15 12:29:04 +0000] [21750] [INFO] Booting worker with pid: 21750
[2021-01-15 12:29:05 +0000] [21574] [INFO] Handling signal: term
[2021-01-15 12:29:05 +0000] [21750] [INFO] Worker exiting (pid: 21750)
[2021-01-15 12:29:05 +0000] [21749] [INFO] Worker exiting (pid: 21749)
[2021-01-15 12:29:05 +0000] [21574] [INFO] Shutting down: Master
[2021-01-15 12:29:12 +0000] [21785] [INFO] Starting gunicorn 19.10.0
[2021-01-15 12:29:12 +0000] [21785] [INFO] Listening at: http://127.0.0.1:8000 (21785)
[2021-01-15 12:29:12 +0000] [21785] [INFO] Using worker: sync
[2021-01-15 12:29:12 +0000] [21821] [INFO] Booting worker with pid: 21821
[2021-01-15 12:29:12 +0000] [21822] [INFO] Booting worker with pid: 21822
[2021-01-15 12:39:34 +0000] [21785] [INFO] Handling signal: term
[2021-01-15 12:39:34 +0000] [21821] [INFO] Worker exiting (pid: 21821)
[2021-01-15 12:39:34 +0000] [21822] [INFO] Worker exiting (pid: 21822)
[2021-01-15 12:39:35 +0000] [21785] [INFO] Shutting down: Master
[2021-03-13 06:03:03 +0000] [2097] [INFO] Starting gunicorn 19.10.0
[2021-03-13 06:03:03 +0000] [2097] [INFO] Listening at: http://127.0.0.1:8000 (2097)
[2021-03-13 06:03:03 +0000] [2097] [INFO] Using worker: sync
[2021-03-13 06:03:03 +0000] [2180] [INFO] Booting worker with pid: 2180
[2021-03-13 06:03:03 +0000] [2181] [INFO] Booting worker with pid: 2181
[2021-03-13 06:07:32 +0000] [2097] [CRITICAL] WORKER TIMEOUT (pid:2180)
[2021-03-13 06:07:32 +0000] [2180] [ERROR] Error handling request /api/method/frappe.desk.page.setup_wizard.setup_wizard.setup_complete
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/page/setup_wizard/setup_wizard.py", line 72, in setup_complete
    task.get('fn')(task.get('args'))
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/setup/setup_wizard/setup_wizard.py", line 95, in setup_company
    fixtures.install_company(args)
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/setup/setup_wizard/operations/install_fixtures.py", line 396, in install_company
    make_records(records)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/page/setup_wizard/setup_wizard.py", line 392, in make_records
    doc.insert(ignore_permissions=True)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 260, in insert
    self.run_post_save_methods()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 927, in run_post_save_methods
    self.run_method("on_update")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 797, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 1073, in composer
    return composed(self, method, *args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 1056, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 791, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/setup/doctype/company/company.py", line 107, in on_update
    install_country_fixtures(self.name)
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/setup/doctype/company/company.py", line 419, in install_country_fixtures
    frappe.get_attr(module_name)(company_doc, False)
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 14, in setup
    setup_company_independent_fixtures()
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 21, in setup_company_independent_fixtures
    make_custom_fields()
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 530, in make_custom_fields
    create_custom_fields(custom_fields, update=update)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 143, in create_custom_fields
    create_custom_field(doctype, df, ignore_validate=ignore_validate)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 127, in create_custom_field
    custom_field.insert()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 230, in insert
    self.run_before_save_methods()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 896, in run_before_save_methods
    self.run_method("validate")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 797, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 1073, in composer
    return composed(self, method, *args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 1056, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 791, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 35, in validate
    meta = frappe.get_meta(self.dt, cached=False)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 787, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/meta.py", line 43, in get_meta
    return load_meta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/meta.py", line 46, in load_meta
    return Meta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/document.py", line 162, in load_from_db
    "*", as_dict=True, order_by="idx asc")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 445, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 592, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 156, in sql
    self._cursor.execute(query, values)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1082, in read
    self._read_result_packet(first_packet)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1152, in _read_result_packet
    self._read_rowdata_packet()
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1190, in _read_rowdata_packet
    rows.append(self._read_row_from_packet(packet))
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1209, in _read_row_from_packet
    data = converter(data)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/data.py", line 69, in get_datetime
    return datetime.datetime.strptime(datetime_str, DATETIME_FORMAT)
  File "/usr/lib/python3.6/_strptime.py", line 565, in _strptime_datetime
    tt, fraction = _strptime(data_string, format)
  File "/usr/lib/python3.6/_strptime.py", line 334, in _strptime
    if (_getlang() != locale_time.lang or
  File "/usr/lib/python3.6/_strptime.py", line 31, in _getlang
    return locale.getlocale(locale.LC_TIME)
  File "/home/frappe/frappe-bench/env/lib/python3.6/locale.py", line 581, in getlocale
    return _parse_localename(localename)
  File "/home/frappe/frappe-bench/env/lib/python3.6/locale.py", line 476, in _parse_localename
    code = normalize(localename)
  File "/home/frappe/frappe-bench/env/lib/python3.6/locale.py", line 419, in normalize
    code = locale_alias.get(lookup_name, None)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/gunicorn/workers/base.py", line 202, in handle_abort
    sys.exit(1)
SystemExit: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/werkzeug/local.py", line 231, in application
    return ClosingIterator(app(environ, start_response), self.cleanup)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/werkzeug/wrappers/base_request.py", line 237, in application
    resp = f(*args[:-2] + (request,))
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 92, in application
    frappe.db.rollback()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 753, in rollback
    self.sql("rollback")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 171, in sql
    self._cursor.execute(query)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/home/frappe/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 671, in _read_packet
    % (packet_number, self._next_seq_id))
pymysql.err.InternalError: Packet sequence number wrong - got 195 expected 1
[2021-03-13 06:07:32 +0000] [2180] [INFO] Worker exiting (pid: 2180)
[2021-03-13 06:07:33 +0000] [2527] [INFO] Booting worker with pid: 2527
[2021-03-13 08:08:19 +0000] [2097] [INFO] Handling signal: term
[2021-03-13 08:08:19 +0000] [2181] [INFO] Worker exiting (pid: 2181)
[2021-03-13 08:08:19 +0000] [2527] [INFO] Worker exiting (pid: 2527)
[2021-03-13 08:08:20 +0000] [2097] [INFO] Shutting down: Master
